lib/grizzly/zipgateway/supervisor.ex

```diff
@@ -102,7 +102,15 @@ defmodule Grizzly.ZIPGateway.Supervisor do
       type: :supervisor,
       start:
         {Supervisor, :start_link,
-         [children, [name: Grizzly.ZIPGateway.ProcessSupervisor, strategy: :rest_for_one]]}
+         [
+           children,
+           [
+             name: Grizzly.ZIPGateway.ProcessSupervisor,
+             strategy: :rest_for_one,
+             max_restarts: 1,
+             max_seconds: 1
```

Comment:
This doesn't do what it sounds like it should. What it says is that the supervisor only allows for 1 restart in 1 second. If zipgateway were to exit a second time in a second, then it wouldn't be restarted again.